{
	"vulinfoid": "JVNDB-2022-016252",
	"vulinfodata": {
		"title": "SQLAlchemy の mako 他複数ベンダの製品における非効率的な正規表現の複雑さに関する脆弱性",
		"vulinfodescription": {
			"overview": "SQLAlchemy の mako 他複数ベンダの製品には、非効率的な正規表現の複雑さに関する脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Debian",
					"productname": "Debian GNU/Linux",
					"cpe": {
						"text": "cpe:/o:debian:debian_linux",
						"version": "2.2"
					},
					"versionnumber": [
						"10.0"
					]
				},
				{
					"name": "SQLAlchemy",
					"productname": "mako",
					"cpe": {
						"text": "cpe:/a:sqlalchemy:mako",
						"version": "2.2"
					},
					"versionnumber": [
						"1.2.2 未満"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "3.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダアドバイザリまたはパッチ情報が公開されています。参考情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2022-40023",
					"url": "https://www.cve.org/CVERecord?id=CVE-2022-40023"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2022-40023",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-40023"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (issues/366)",
					"url": "https://github.com/sqlalchemy/mako/issues/366"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "lists.debian.org (msg00026)",
					"url": "https://lists.debian.org/debian-lts-announce/2022/09/msg00026.html"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "pyup.io (pyup-discovers-redos-vulnerabilities-in-top-python-packages)",
					"url": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "pyup.io (CVE-2022-40023/50870)",
					"url": "https://pyup.io/vulnerabilities/CVE-2022-40023/50870/"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (extract.py#L21)",
					"url": "https://github.com/sqlalchemy/mako/blob/c2f392e0be52dc67d1b9770ab8cce6a9c736d547/mako/ext/extract.py#L21"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "github.com (925760291d6efec64fda6e9dd1fd9cfbd5be068c)",
					"url": "https://github.com/sqlalchemy/mako/commit/925760291d6efec64fda6e9dd1fd9cfbd5be068c"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-1333",
					"url": "https://cwe.mitre.org/data/definitions/1333.html",
					"title": "非効率的な正規表現の複雑さ"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2023-10-03T17:06:34+09:00",
					"description": "[2023年10月03日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2023-10-03T17:06:34+09:00",
		"datelastupdated": "2023-10-03T17:06:34+09:00",
		"datepublic": "2022-09-07T00:00:00+09:00"
	}
}
